src/nav/infrastructure.yml

```diff
@@ -454,8 +454,10 @@ pages:
         pages:
           - title: ALB/NLB integration
             path: /docs/infrastructure/amazon-integrations/aws-integrations-list/aws-albnlb-monitoring-integration
-          - title: API Gateway integration
-            path: /docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-monitoring-integration
+          - title: API Gateway v1 integration
```

Comment:
Change this to title: API Gateway REST (v1) integration

Reply:
@glennthomas1, i made the suggested changes.